src/components/TagSuggestions.tsx

```diff
@@ -0,0 +1,301 @@
+export const MBTTAGS = [
```

Comment:
Thanks :smile: 
I typed it out. I am typing these words since many months everyday !
We can add a few more from the data surely.

Reply:
If we are enforcing lowercase we will probably want to make sure this list is lowercase as well to avoid confusion.

Reply:
Also, I think it is important to mention, the concept of suggesting tags has been discussed several times in the past. I have so far decided not to implement this feature for one main reason. I am concerned that providing suggestions will introduce a bias toward tag names and prevent users from accessing their own intuition/creativity to create the tags they feel most describe the segment since it is much easier to select a tag from a list than to create a new one.

I definitely understand this could help volunteers such as yourself who do many videos, however I am cautious to provide this feature to all users. If we do provide this to volunteers, I think it would be a good idea to encourage them to make create new tags if they don't find a good fit from the suggestions.

Reply:
So now that you have rewritten the component, it seems you have decided to keep the suggestions. Then we are ready to merge this PR.
Anyway, if you like we can have a toggle switch to disable the suggestions and keep it off by default.